Examples/Scripts/Python/material_validation.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import os
+
+from acts.examples import Sequencer, RootMaterialTrackWriter
+
+import acts
+from acts import (
+    UnitConstants as u,
+)
+
+from common import getOpenDataDetector
+
+
+def runMaterialValidation(
+    trackingGeometry,
+    decorators,
+    field,
+    outputDir,
+    s=None,
+):
+    s = s or Sequencer(events=1000, numThreads=-1)
+
+    for decorator in decorators:
+        s.addContextDecorator(decorator)
+
+    nav = acts.Navigator(trackingGeometry=trackingGeometry)
+
+    stepper = acts.StraightLineStepper()
+    # stepper = acts.EigenStepper(field)
+
+    prop = acts.examples.ConcretePropagator(acts.Propagator(stepper, nav))
+
+    rnd = acts.examples.RandomNumbers(seed=42)
+
+    alg = acts.examples.PropagationAlgorithm(
+        propagatorImpl=prop,
+        level=acts.logging.INFO,
+        randomNumberSvc=rnd,
+        ntests=1000,
+        sterileLogger=True,
+        propagationStepCollection="propagation-steps",
+        recordMaterialInteractions=True,
+    )
+
+    s.addAlgorithm(alg)
+
+    s.addWriter(
+        RootMaterialTrackWriter(
+            level=acts.logging.INFO,
+            collection=alg.config.propagationMaterialCollection,
+            filePath=os.path.join(
+                outputDir, f"{alg.config.propagationMaterialCollection}.root"
```

Comment:
Changed back to the old name to work with the validation root script.
```suggestion
            filePath=os.path.join(
                outputDir, "propagation-material.root"
```